Close #819

kskinoue0612 opened the pull request:
Please make sure the followings:
* For "Elevation" in "Geographic data", "\*.dat (RIC-Nays DEM)", "\*,adf (USGS DEM)" and "\*.anc" are removed from "Import" menu file type combo box.
* When \*.tpo file is imported, "\*.dat (RIC-Nays DEM)" is removed from "Export" file type combo box.
* You can import from various types of text files, with the setting dialog below:

![screenshot](https://user-images.githubusercontent.com/4031569/95869379-42b7b500-0da6-11eb-9bea-69377558613d.png)

You can test with the files in issue-819.zip.

[issue-819.zip](https://github.com/i-RIC/prepost-gui/files/5371731/issue-819.zip)

scharlton2 commented:
Hi Keisuke,

I corrected some of the English grammar in 2397bb2.  I canceled the appveyor build.

Some of the imports didn't work the way I expected them to.

The files that began with:
THIS IS A TEST FILE FOR IMPORTING
didn't parse the values on the remaining lines unless "Space" was selected even when ignoring header lines:

Case 1: File: test_comma.txt (with Comma and Space delimiters)
![image](https://user-images.githubusercontent.com/7538707/96326722-5a827980-0ff0-11eb-8c30-4b8fe55eec5a.png)

Case 2: File: test_comma.txt (with just Comma delimiters)
![image](https://user-images.githubusercontent.com/7538707/96326732-72f29400-0ff0-11eb-97bb-29c96b5ef666.png)

Case 3: File: test_comma.txt (with Comma delimiters and ignoring the top two lines:)
![image](https://user-images.githubusercontent.com/7538707/96326775-ce248680-0ff0-11eb-8ad0-ffd8078dc7dd.png)


I think that at least Case 3 should work without selecting the Space delimiter.
Here's how Microsoft Excel works:
![image](https://user-images.githubusercontent.com/7538707/96326944-40499b00-0ff2-11eb-8963-efb3a543632d.png)


You also might want to add the file name some where in the dialog, I kept forgetting which file I was importing.

Thanks,
Scott

kskinoue0612 commented:
Hi Scott,

Thank you for checking. I think I can work on this in a few days.
I agree that I should add the file name on the dialog, and I'll work on that too.

Keisuke

kskinoue0612 commented:
Hi Scott,

I've fixed the issue you've reported, and added function to show file name on the dialog.
I've also fixed the caption for importer from "CSV, text" to "Text files (CSV, TSV, etc.)" so that it is easier to understand.
Thanks,

Keisuke
